<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инспектор аэропортов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resursy.qrc">
    <normaloff>:/Airplane.ico</normaloff>:/Airplane.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="gB_statistics">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Просмотр статистики загруженности аэропорта</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pB_statistics">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Просмотреть статистику</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pB_search">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="gB_list_airports">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>205</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>205</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Аэропорт</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="cB_list_airports">
         <property name="minimumSize">
          <size>
           <width>185</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>185</width>
           <height>22</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertPolicy::NoInsert</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon</enum>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="gB_list_flights">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>350</height>
       </size>
      </property>
      <property name="title">
       <string>Список рейсов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="gB_date_flight">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Дата рейса</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDateEdit" name="dateE_flight">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2016</year>
           <month>8</month>
           <day>15</day>
          </datetime>
         </property>
         <property name="maximumDate">
          <date>
           <year>2017</year>
           <month>9</month>
           <day>14</day>
          </date>
         </property>
         <property name="minimumDate">
          <date>
           <year>2016</year>
           <month>8</month>
           <day>15</day>
          </date>
         </property>
         <property name="minimumTime">
          <time>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::Section::DaySection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gB_flight_direction">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Направление рейсов</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rB_arrival">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Прилёт</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rB_departure">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Вылет</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cB_list_airports</tabstop>
  <tabstop>rB_arrival</tabstop>
  <tabstop>rB_departure</tabstop>
  <tabstop>dateE_flight</tabstop>
  <tabstop>pB_search</tabstop>
  <tabstop>pB_statistics</tabstop>
 </tabstops>
 <resources>
  <include location="resursy.qrc"/>
 </resources>
 <connections/>
</ui>
